### PR TITLE
Include necessary data for simulations within the auction json sent to solvers

### PR DESCRIPTION
**Pull request by Mateo-mro**
Include necessary data for simulations within the auction json sent to solvers. Follow up for https://github.com/cowprotocol/services/pull/2620

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -516,12 +566,6 @@ components:
           $ref: "#/components/schemas/SigningScheme"
         signature:
           $ref: "#/components/schemas/Signature"
-        feePolicies:
```
Comment (Mateo-mro) [inline openapi.yml:519]: @MartinquaXD I think this was incorrect from https://github.com/gnosis/solvers/pull/8

**Comment by Mateo-mro** [comment]
Note: I didn't add those fields to `required` because otherwise it will crash until the change hits production. This way we can merge this as it is to avoid errors with the new changes, and after the changes are in prod, we can make them `required`.

File: openapi.yml (hunk 1 above)
Comment (MartinquaXD) [reply openapi.yml:519]: Why is that? With the introduction of the `fee-handler` parameter these policies can now be forwarded to the solver, no?

File: openapi.yml [hunk 2]
```diff
@@ -208,19 +208,69 @@ components:
...
+        owner:
+          $ref: "#/components/schemas/Address"
         partiallyFillable:
           description: |
             Whether or not this order can be partially filled. If this is false,
             then the order is a "fill-or-kill" order, meaning it needs to be
             completely filled or not at all.
           type: boolean
+        executed:
+          $ref: "#/components/schemas/TokenAmount"
```
Comment (MartinquaXD) [inline openapi.yml:238]: Redundant

**Review by MartinquaXD — COMMENTED** [review]
> Note: I didn't add those fields to required because otherwise it will crash until the change hits production.

The openapispec has no influence on the runtime behavior of the code. The pods will crashloop until the `solvers` update their `solvers-dto` dependency after the main PR in the backend was merged.

Will review again once the main PR is ready for merge because splitting discussions over 2 repos doesn't seem like a good idea.

File: openapi.yml (hunk 1 above)
Comment (Mateo-mro) [reply openapi.yml:519]: Yes, but this is the `JitOrder`. The `feePolicies` are forwarded only for `Orders`, as I added [here](https://github.com/gnosis/solvers/pull/18/files#diff-5016bdc926b6c0f649f363739d09159a1db664aae19a08fa6b38b03406f040dcR217), or maybe am I missing something? :thinking:

Comment (MartinquaXD) [reply openapi.yml:519]: Oh, didn't notice it was on the JIT order. Then it's indeed a mistake. 👍

**Review by squadgazzz — APPROVED** [review]

**Pull request merged**